packages/database/package.json

```diff
@@ -11,7 +11,7 @@
   "author": "Beyond Essential Systems <admin@tupaia.org> (https://beyondessential.com.au)",
   "main": "dist/index.js",
   "scripts": {
-    "build": "babel src --out-dir dist --source-maps --ignore \"**/migrations\",\"src/tests/**\" --config-file \"../../babel.config.json\" && tsc",
+    "build": "babel src --out-dir dist --source-maps --config-file \"../../babel.config.json\" && tsc",
```

Comment:
I think we should still ignore `src/tests` when building

Reply:
They should get ignored by the dynamic rule within the .babelrc.js :-)

Reply:
Oh yes you are right, I read this file first 👍